src/sentry/replays/endpoints/project_replay_viewed_by.py

```diff
@@ -104,8 +110,46 @@ def post(self, request: Request, project: Project, replay_id: str) -> Response:
         except ValueError:
             return Response(status=404)
 
-        # Synchronously publish a replay-viewed event.
         message = viewed_event(project.id, replay_id, request.user.id)
         publish_replay_event(message, is_async=False)
 
         return Response(status=204)
+
+
+def _normalize_user(user: dict[str, Any]) -> dict[str, Any]:
+    """Return a normalized user dictionary.
+
+    The viewed-by resource is expected to return a subset of the user_service's
+    response output.
+    """
+    return {
```

Comment:
What are the chances the output of user_service.serialize changes, and this breaks? Should we use .get() instead?